plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -281,13 +556,21 @@ export const resolvers = (): IResolvers => ({
     ): Observable<JobConnection | null> {
       return Observable.of(null);
     },
-    job(_obj = {}, _args: GeneralArgs, _context = {}): Observable<Job | null> {
-      return Observable.of(null);
+    job(_obj = {}, args: GeneralArgs, _context = {}): Observable<Job | null> {
+      // TODO: Write typeguard here
+      const { id } = args as JobQueryArgs;
+      const pollingInterval$ = Observable.timer(0, pollingInterval);
+      const responses$ = pollingInterval$.switchMap(() => fetchJobDetail(id));
+
+      return responses$.map(response => typeResolvers.Job(response));
     }
   }
 });
 
 export default makeExecutableSchema({
   typeDefs,
-  resolvers: resolvers()
+  resolvers: resolvers({
```

Comment:
I fix that on my PR, so I would say it is ok, but otherwise:

```
  resolvers: resolvers({
    fetchJobDetail: MetronomeClient.fetchJobDetail,
    pollingInterval: Config.getRefreshRate()
  })
```